app/scripts/modules/amazon/src/help/amazon.help.ts

```diff
@@ -101,7 +101,7 @@ const helpContents: { [key: string]: string } = {
   'aws.targetGroup.port':
     'The port on which the targets receive traffic. Cannot be edited after being saved; if you want to use a different port, create a new target group, save the load balancer, move your targets, and then delete this target group.',
   'aws.targetGroup.attributes.deregistrationDelay':
-    'The amount of time for Elastic Load Balancing to wait before changing the state of a deregistering target from draining to unused. The range is 0-3600 seconds. The default value is 300 seconds.',
+    'The amount of time for Load Balancing to wait before changing the state of a deregistering target from draining to unused. The range is 0-3600 seconds. The default value is 300 seconds.',
```

Comment:
at this point, might as well just change it to `the load balancer`

Reply:
changed